deploy/local/docker-compose.yml

```diff
@@ -173,6 +174,100 @@ services:
     depends_on:
       - postgres
 
+  ragflow:
+    image: infiniflow/ragflow:v0.21.1-slim
+    platform: linux/amd64
+    container_name: custom-rag-ragflow
+    restart: on-failure
+    depends_on:
+      ragflow-es:
+        condition: service_healthy
+      minio:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+      ragflow-tei:
+        condition: service_started
+      postgres:
+        condition: service_healthy
+      ragflow-db-init:
+        condition: service_completed_successfully
+    ports:
+      - "7080:80"
+      - "7443:443"
+      - "9380:9380"
+      - "9381:9381"
+      - "9382:9382"
+    env_file:
+      - "config/.env.ragflow"
+    volumes:
+      - custom-rag-ragflow-logs:/ragflow/logs
+      - ./ragflow/service_conf.yaml.template:/ragflow/conf/service_conf.yaml.template
+      - ./ragflow/entrypoint.sh:/ragflow/entrypoint.sh
+      - ./ragflow/history_data_agent:/ragflow/history_data_agent
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  ragflow-es:
+    image: elasticsearch:8.11.3
+    container_name: custom-rag-ragflow-es
+    restart: on-failure
+    env_file: "config/.env.ragflow"
+    environment:
+      - node.name=ragflow-es
+      - ELASTIC_PASSWORD=${ELASTIC_PASSWORD:-ragflow_elastic_pass}
+      - bootstrap.memory_lock=false
+      - discovery.type=single-node
+      - xpack.security.enabled=true
+      - xpack.security.http.ssl.enabled=false
+      - xpack.security.transport.ssl.enabled=false
+      - cluster.routing.allocation.disk.watermark.low=5gb
+      - cluster.routing.allocation.disk.watermark.high=3gb
+      - cluster.routing.allocation.disk.watermark.flood_stage=2gb
+      - ES_JAVA_OPTS=${ES_JAVA_OPTS:--Xms512m -Xmx512m}
+    mem_limit: 4g
+    ports:
+      - "9202:9200"
+    volumes:
+      - custom-rag-ragflow-es-data:/usr/share/elasticsearch/data
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s http://localhost:9200 >/dev/null"]
+      interval: 10s
+      timeout: 10s
+      retries: 120
+
+  ragflow-tei:
```

Comment:
Why do we need to have another model hosting service? I don't think we want to add model hosting (especially on a CPU) to this project.